### PR TITLE
Color token changes for V2

### PR DESCRIPTION
**Pull request by kalpeshp0310**
Add colours based on new color scheme.

New color scheme is almost same as the old one. Just couple of noticeable difference.
1. We do not ask for 10,20,30, etc alpha variants of colors from user. We compute it in our library.
2. surface scrim and background scrims have been added to colors.

### CODE REVIEW

File: example/lib/styles/colors.dart [hunk 1]
```diff
@@ -87,37 +75,44 @@ class AppColors {
   static const Color inputTextDimDark = Color(0xB2FCFCFD);
 
   // Utility
-  static const Color disabledBackground = Color(0x0D1A1B1F);
+  static const Color disabledBackground = Color.fromARGB(31, 26, 27, 31);
```
Comment (Ibrahimhass) [inline example/lib/styles/colors.dart:78]: @kalpeshp0310 Please use consistent formatting, as in all other places.
The  31 value can be converted to hex.

File: example/lib/styles/colors.dart [hunk 2]
```diff
@@ -146,6 +141,6 @@ class AppColors {
   static const Color warningHover = Color(0xffFFAD14);
   static const Color warningHoverDark = Color(0xffFFA90A);
 
-  static const Color disabledBackgroundHover = Color(0x331A1B1F);
+  static const Color disabledBackgroundHover = Color.fromARGB(51, 26, 27, 31);
```
Comment (Ibrahimhass) [inline example/lib/styles/colors.dart:144]: Same as earlier 51 can be converted to hex and the format used in other places can be used for consistency.

Comment (kalpeshp0310) [reply example/lib/styles/colors.dart]: Done.

File: example/lib/styles/colors.dart [hunk 3]
```diff
@@ -29,41 +29,29 @@ class AppColors {
...
-  static const Color warning10 = Color(0x1AFFB938);
-  static const Color warning10Dark = Color(0x1AFFB938);
-
-  static const Color error10 = Color(0x1ACD1D32);
-  static const Color error10Dark = Color(0x1AD42136);
-
   // On Colors
-  static const Color onPrimary = Color(0xffF5F8FF);
+  static const Color onPrimary = Color(0xffF5F9FF);
   static const Color onPrimaryDark = Color(0xffFAFCFF);
```
Comment (Ibrahimhass) [inline example/lib/styles/colors.dart:34]: @kalpeshp0310 I notice that the color values for the dark theme have not changed.
I used these [design](https://www.figma.com/file/pxXcMnd1EtFLny3NQy1s8R/Tarka-kit--Dark-mode-(For-devs)?type=design&node-id=1220%3A142028&mode=design&t=bedlCVeTFqLndPzs-1) to set the color values for dark mode.

Comment (kalpeshp0310) [reply example/lib/styles/colors.dart:34]: I don't think we need to change it. The light variant of same design had value change.
I think we are good here.

**Review by Ibrahimhass — APPROVED** [review]

**Pull request merged**